aws_doc_sdk_examples_tools/stats.py

```diff
@@ -20,6 +21,9 @@ def main(roots: List[str]):
         print(f"Snippets	{stats['snippets']}")
         genai = pformat(dict(stats["genai"]))
         print(f"GenAI	{genai}")
+        stats['root'] = docgen_root.name
+        all_stats.append(stats)
+    return all_stats
```

Comment:
Why are you returning a list from a `main`? What's the goal for this change?

Reply:
I want stats.py to return the stats, so it can be used like this: `stats = doc_gen.stats()`.